docs/getting-started/integrations-cdn/react-multiroot-cdn.md

```diff
@@ -38,50 +38,131 @@ npm install ckeditor5 @ckeditor/ckeditor5-react
 Use the `useMultiRootEditor` hook inside your project:
 
 ```tsx
```

Comment:
In every other snippet we use `tab` instead of `spaces` for indent, here are spaces, is this an intentional?